tests/integration/cairo1_hint_locals/prepare/prepare_test.cairo

```diff
@@ -23,3 +23,43 @@ fn test_prepare_with_args() {
     calldata.append('name');
     drop(prepare(class_hash, calldata).unwrap());
 }
+
+#[test]
+fn test_prepare_cairo0() {
+    let class_hash = declare_cairo0('cairo0').unwrap();
+
+    drop(prepare(class_hash, ArrayTrait::new()).unwrap());
+
+}
+
+#[test]
+fn test_prepare_cairo0_w_ctor() {
+    let class_hash = declare_cairo0('cairo0_w_ctor').unwrap();
+
+    let mut calldata = ArrayTrait::new();
+    calldata.append(200);
+    drop(prepare(class_hash, calldata).unwrap());
+
+}
+
+#[test]
+fn test_prepare_cairo0_w_ctor_no_args() {
+    let class_hash = declare_cairo0('cairo0_w_ctor_no_args').unwrap();
+
+    drop(prepare(class_hash, ArrayTrait::new()).unwrap());
+}
+
+use serde::Serde;
+use minimal_custom_structures::MinimalContract::CustomStruct;
+use minimal_custom_structures::MinimalContract::CustomStructSerde;
+
+
+#[test]
+fn test_custom_structures_usage() {
+    let class_hash = declare('minimal_custom_structures').unwrap();
+    let mut args = ArrayTrait::new();
+    let cs = CustomStruct { a: 1, b: 2 };
+    CustomStructSerde::serialize(ref args, cs);
+
+    drop(prepare(class_hash, args).unwrap());
```

Comment:
What does this check exactly? In the end, we have an array passed as an argument anyway.

Reply:
Yes that's right, this only tests if it's doable